lxd/instance_snapshot.go

```diff
@@ -524,10 +524,10 @@ func snapshotPut(s *state.State, r *http.Request, snapInst instance.Instance) re
 
 	opType := operationtype.SnapshotUpdate
 
-	instName, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
+	parentName, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
```

Comment:
Please can you rebase this change so that within the same PR we don't see this line change from:

Added in 4f2ed1ddc8ba4ba1e6aef4e45bca71d2c0ddfabb
```
instName, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
```

to 


Changed in 6b181e1b85be526cc830672f6a40789cc7a4dfb8
```
parentName, _, _ := api.GetParentAndSnapshotName(snapInst.Name())
```

Reply:
Sure, I squashed the commits together so that the history becomes clean again.

Reply:
OK thanks. Although thats not quite what I meant.

I meant that each logical change should still be in its own commit, but that multiple commits within the same PR shouldn't add a line and then "fix" that line later.

Its fine as it is now though as its a small PR anyway, but for larger PRs, keep in mind we dont do squash merges, we do keep the commit history from the PR into the main branch, so its important to separate your commits logically.

This helps us cherry-pick commits for backporting into the LTS branch.

Reply:
For example I would have broken this PR down into the following commits:

- Removing the containerName argument to functions and instead extracting parent inside those functions where used.
- Fixing the resources URL (which fixes the particular issue).
- Correcting `inst` to `snapInst` var.

Reply:
Thanks, will adapt this pattern for the next changes.